src/libraries/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -423,7 +423,7 @@ public static IEnumerable<Object[]> SupportedProtocolData()
                     await clientAuthentication.WaitAsync(TestConfiguration.PassingTestTimeout);
                     _logVerbose.WriteLine("ServerAsyncAuthenticateTest.clientAuthentication complete.");
                 }
-                catch (Exception ex)
+                catch (Exception ex) when (ex is AuthenticationException)
```

Comment:
Why not `catch (AuthenticationException ex)`?

Reply:
yes, that make sense. updated.